libs/dao-data/src/Query.ts

```diff
@@ -317,7 +320,7 @@ export default class Query {
     }
   }
 
-  public async findLatestTransaction({
+  public async findLatestTransactionByDao({
```

Comment:
Should we allow clients to pass in the where clause?

Reply:
Yeah that's fair. this was a super scoped function for post proposal tx query... but was anticipating a usecase we might not need.

going to convert to a more typical list query and we can pass the where, sort order and limit like this is doing.

Reply:
@alexkeating  new commit has this. much cleaner and more flexible. thanks!